### ISSUE TITLE
Track back with files path

### ISSUE THREAD

**Issue by Vieiraork** [TaylorSMarks/playsound#85]
Hello,

I have so much problems with last version of packet. The error in reconize path file contains 'backslash' this simbol, I use python version 3.8.2 now using version 1.2.2 of playsound. Sorry for English but I hope can help others with same problem.

In file I use glob packege to return a list files 'Music' folder on Windows but in last version always return erros, but in version 1.2.2 this erros not occur.

Code used in test is here https://pastebin.com/u1MWr5DY.
[error.txt](https://github.com/TaylorSMarks/playsound/files/7002408/error.txt)

**Comment by makwadajp** [comment]
I also get the the following error along with `Error 263` with MCI device issues, as mentioned in your above [error.txt](https://github.com/TaylorSMarks/playsound/files/7002408/error.txt) link on Windows. Furthermore, this issue might be related to [Error 263 and 305](https://github.com/TaylorSMarks/playsound/issues/98) (**[my comment](https://github.com/TaylorSMarks/playsound/issues/98#issuecomment-929092057)**) which seems to be happening on wav files on Windows when directly providing audio's file path (while I understand your code uses **glob** for **mp3** files).

```
  File "C:\Users\mwada\Anaconda3\envs\test\lib\site-packages\playsound.py", line 72, in _playsoundWin
    winCommand(u'open {}'.format(sound))
  File "C:\Users\mwada\Anaconda3\envs\test\lib\site-packages\playsound.py", line 64, in winCommand
    raise PlaysoundException(exceptionMessage)
```

**Comment by Vieiraork** [comment]
Yeah, I'm use glob module, but I dont understand the fact of the error occours with mp3 files also. Thank you for response!
